dev/breeze/doc/05_test_commands.rst

```diff
@@ -218,6 +218,29 @@ Here is the detailed set of options for the ``breeze testing providers-integrati
   :alt: Breeze testing providers-integration-tests
 
 
+Running OpenAPI Client tests
```

Comment:
Possibly we should change it to "Python API client tests" here and in CI jobs. It's more accurate description.

Reply:
Make sense..